src/library/rhythmbox/rhythmboxfeature.h

```diff
@@ -18,44 +20,46 @@ class BaseExternalPlaylistModel;
 class RhythmboxFeature : public BaseExternalLibraryFeature {
     Q_OBJECT
  public:
```

Comment:
It looks like this indentation has tricked clang-format. 
Here we use two space indent. 
The rest should have four spaces as before.